bmarlow/Week 3/mailroom.py

```diff
@@ -0,0 +1,131 @@
+import os
+
+#build our initial dictionary
+dictionary = {
+	'Ricky': '1000',
+	'Bobby': '1100,2200',
+	'Hansel': '5000,6000,7000',
+	'Mr. Burgandy': '1,2,3,4',
+	'Austin': '123,4567,78,99,100',
+	'Dale': '14,234,878,345,345,234'
+}
```

Comment:
Might want to consider giving the dictionary a more descriptive name.  If someone is reading code 60 lines down and sees a reference to it, they won't know what it refers to or what data it might hold.